Specs/Scene/ModelExperimental/ModelSilhouettePipelineStageSpec.js

```diff
@@ -8,10 +8,16 @@ import {
 import ShaderBuilderTester from "../../ShaderBuilderTester.js";
 
 describe("Scene/ModelExperimental/ModelSilhouettePipelineStage", function () {
+  beforeAll(function () {
```

Comment:
I would change this to `beforeEach()` just to avoid surprises if we ever have to add more test cases